### ISSUE TITLE
npm ERR! peerinvalid error on package zone.js@0.7.2 prevents installation of angular2-tree-component@2.5.0 with @angular/core@2.3.0

### ISSUE THREAD

**Issue by KeithGillette** [500tech/angular2-tree-component#125]
I'd love to try out this treeview, but am unable to install in my project running the latest version of AngularJS 2, v2.3.0, which has a peer dependency on zone.js^0.7.2. However, due to npm's handling of sub-version 1.0 releases, this results in an error preventing installation given angular2-tree-component 2.5.0's peer dependency on zone.js ^0.6.21:

```
6272 error code EPEERINVALID
6273 error peerinvalid The package zone.js@0.7.2 does not satisfy its siblings' peerDependencies requirements!
6273 error peerinvalid Peer @angular/core@2.3.0 wants zone.js@^0.7.2
6273 error peerinvalid Peer angular-in-memory-web-api@0.1.17 wants zone.js@^0.7.2
6273 error peerinvalid Peer angular2-tree-component@2.5.0 wants zone.js@^0.6.21
```

Is this a true incompatibility or is there an installation work-around? Thanks!

**Comment by pixelchutes** [comment]
Also wondering the same. Any reason this can't work against `zone.js@0.7.2`?

**Comment by KeithGillette** [comment]
It looks like @schmuli's open [Pull request](https://github.com/500tech/angular2-tree-component/pull/126) addresses this issue. I have downloaded, built, and installed his fork and can verify it works with Angular 2.3.0 and zone.js 0.7.2 (even embedded inside Angular 1.5.8 directive component into which the Angular2 child component is downgraded using UpgradeModule).

**Comment by adamkleingit** [comment]
I'll merge @schmuli 's PR very soon.

**Comment by tomalex0** [comment]
thanks  @adamkleingit

**Comment by adamkleingit** [comment]
Fixed in 2.5.1 thanks to @schmuli 's PR #126 
Can you verify?
Thanks!

**Comment by KeithGillette** [comment]
I confirm that 2.5.1 fixes this installation issue for me. Thanks!
